sdk/python/kfp/dsl/__init__.py

```diff
@@ -15,13 +15,14 @@
 
 from ._pipeline_param import PipelineParam, match_serialized_pipelineparam
 from ._pipeline import Pipeline, pipeline, get_pipeline_conf
-from ._container_op import ContainerOp, UserContainer, Sidecar
+from ._container_op import ContainerOp, UserContainer, Sidecar, BaseOp
```

Comment:
WHy do you think we should expose BaseOp to the users? I'm not sure they're expected to instantiate it.

Reply:
leftover from previous hacking